.github/workflows/post-release.yml

```diff
@@ -125,6 +125,10 @@ jobs:
       run: |
         tag="${{ needs.create-release.outputs.tag }}"
         gh release upload "$tag" ${{ env.ASSET }}
+    - name: Update major tag
+      uses: rickstaa/action-update-semver@v1
+      with:
+        major_version_tag_only: true
```

Comment:
This needs to be in the `publish-release` job

Reply:
Moved it to the end of the `publish-release` job.